Lib/test/test_complex.py

```diff
@@ -412,6 +412,18 @@ def __complex__(self):
             self.assertEqual(complex(complex1(1j)), 2j)
         self.assertRaises(TypeError, complex, complex2(1j))
 
+    def test___complex__(self):
```

Comment:
I feel like the whole test file needs to be updated and not just a single function since now we have a test file thats part Python 3.8 part Python 3.11. @youknowone what's do we do in these cases?

Reply:
Yes, you are right. But 3.11 is not published and not even stablized yet. We added tests to `extra_tests/snippets` before in similar situation. But I don't have a concrete idea what's the best.

Reply:
In my opinion, generally we'd better to keep easy to update `Lib/` from CPython. But adding test test here will increase the chance for conflict in Future. So I think adding to `extra_tests` is better than adding here.

Reply:
Yup that makes sense to me too. @Snowapril move this test into `extra_tests/snippets/complex.py` so we can push this through (similarly for bytes).

Reply:
I applied it! (Also same for bytes). As `extra_tests/snippets/complex.py` doesn't exist I newly created it and pushed test.
Thanks :)

Reply:
Uhm... As `__complex__` and `__bytes__` are not a spec of cpython 3.8, command `pipenv run pytest -v` in github action(Run snippets and cpython tests) go failed. How can I handle this case?

Reply:
Ah! I found `skip_if_unsupported` in test utils can resolve this failure.